uber/tests/models/test_promo_code.py

```diff
@@ -0,0 +1,66 @@
+from uber.tests import *
+
+class TestPromoCodeAdjustments:
```

Comment:
Whoops, I started writing these tests and forgot to actually run them. No idea if they work.